frontend/public/components/cluster-settings/cluster-operator.tsx

```diff
@@ -124,17 +127,23 @@ const filters = [{
   })),
 }];
 
-export const ClusterOperatorStartGuide: React.SFC<{}> = () =>
-  <React.Fragment>
-    <h4>What are Cluster Operators?</h4>
-    <p>
-      An Operator is a method of packaging, deploying, and managing a Kubernetes application. Cluster Operators implement and automate updates of OpenShift and Kubernetes at the cluster level. During an update, the latest versions of the OpenShift and Kubernetes components are downloaded. A rolling update will occur to install the latest versions.
-    </p>
-  </React.Fragment>;
+const UpdateInProgressAlert: React.SFC<UpdateInProgressAlertProps> = ({cv}) => {
+  const updateCondition = getClusterVersionCondition(cv, ClusterVersionConditionType.Progressing, K8sResourceConditionStatus.True);
+  return (
+    <React.Fragment>
+      { updateCondition && <div className="co-m-pane__body co-m-pane__body--alt">
+        <div className="alert alert-info">
```

Comment:
Alerts are changing in #1763. We might want to just use the new PF4 style.

cc @rebeccaalpert